src/filter.jl

```diff
@@ -183,7 +183,7 @@ end
 function solν(t,T,B,β,νT)
   #TODO
   # β time-dependent
-  νT*Φfunc(t,T,-B) + Φfunc(t,T,-B)*β*inv(B)*(Φfunc(t,T,B)-1)
+  Φfunc(t,T,B)*(νT + inv(B)*β) - inv(B)*β
```

Comment:
I would expect
```suggestion
  Φfunc(t,T,B)*(νT - inv(B)*β) + inv(B)*β
```

Reply:
I just checked with Mathematica that
```Mathematica
DSolve[nu'[t] == B*nu[t] + beta, nu[t], t]
```
```
{{nu[t] -> -(beta/B) + E^(B t) C[1]}}
```
and then 
`nuT = E^(B T) C[1]- (beta/B)` , so
`C[1] = E^(-B T)(nuT + beta/B)`

Reply:
Ok 😃